developers/weaviate/configuration/replication.md

```diff
@@ -51,6 +51,36 @@ import ReplicationConfigWithAsyncRepair from '/_includes/code/configuration/repl
 
 <ReplicationConfigWithAsyncRepair />
 
+### Configuring async replication
+
+:::info Added in `v1.29`
+Async replication support has been added in `v1.26`while the [environment variables](/developers/weaviate/config-refs/env-vars#multi-node-instances) for configuring async replication (`ASYNC_*`) have been introduced in `v1.29`.
+:::
+
+Async replication is a background synchronization process in Weaviate that ensures eventual consistency across nodes storing the same shard. When a collection is partitioned into multiple shards, each shard is replicated across several nodes (as defined by the replication factor `REPLICATION_MINIMUM_FACTOR`). Async replication guarantees that all nodes holding the same shard remain in sync by periodically comparing and propagating data.
+
+#### 1. Periodic data comparison
```

Comment:
I like the explanation overall. 

One question that I have is whether this could be phrased more like a how-to guide. As currently, written, I wonder if it reads more like an explanation, rather than a guide on how to do things. 

Ultimately the reader can figure it out by inference, but I wonder if we can make the link more explicit. Wdyt?

Reply:
Yep, good point. I will try to make it look more like a "how-to" guide.